documentation/asciidoc/topics/ref_server_configuration_values.adoc

```diff
@@ -0,0 +1,74 @@
+[id='server-configuration-values_{context}']
+= {brandname} Server configuration values
+
+[role="_abstract"]
+{brandname} Server configuration values let you do the following:
+
+* Create caches at startup with any {brandname} configuration.
+* Add cache templates and use them to create caches on demand.
+* Specify one of the default JGroups stacks.
+* Define {brandname} Server security realms.
+* Configure {brandname} Server endpoints with Hot Rod and REST connectors.
+
+{brandname} Server configuration includes the `cacheContainer` fields in addition to `interfaces`, `socketBindings`, `endpoints`, and `security` fields.
+
+[IMPORTANT]
+====
+You must always provide a complete {brandname} Server configuration when you modify `deploy.infinispan` values.
+====
+
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/deploy_infinispan.yaml[]
+----
+
+[discrete]
+== {brandname} cache configuration
+
+.{brandname} cache
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/cache_manager_caches.yaml[]
+----
+
+.Cache template
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/cache_manager_cache_templates.yaml[]
+----
+
+[discrete]
+== Cluster transport
+
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/jgroups_default.yaml[]
+----
+
+[discrete]
+== {brandname} Server configuration
+
+[IMPORTANT]
+====
+Do not modify or remove the following default configuration to avoid loss of service with your {brandname} cluster:
```

Comment:
"loss of service" is maybe a bit too strong here. These can safely be removed if the user does not require prometheus metrics.

Reply:
I've toned this down a little. Please let me know if you think we need to refine further.